hudi-utilities/src/test/resources/delta-streamer-config/source-jdbc.avsc

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
any reason why the existing `source.avsc` won't work for you? Like to avoid creating new schema if possible

Reply:
Because the data structure of source.avs is too complex, the schema of the database table cannot achieve a similar effect.